src/server/JasmineGraphInstanceService.cpp

```diff
@@ -1484,6 +1758,82 @@ std::string JasmineGraphInstanceService::aggregateCentralStoreTriangles(std::str
 
 }
 
+string JasmineGraphInstanceService::aggregateCompositeCentralStoreTriangles(std::string compositeFileList,
+                                                                            std::string availableFileList) {
+    Utils utils;
+    instance_logger.log("###INSTANCE### Started Aggregating Composite Central Store Triangles","info");
+    std::string aggregatorFilePath = utils.getJasmineGraphProperty("org.jasminegraph.server.instance.aggregatefolder");
+    std::string dataFolder = utils.getJasmineGraphProperty("org.jasminegraph.server.instance.datafolder");
+    map<long, unordered_set<long>> aggregatedCompositeCentralStore;
+
+
+    std::vector<std::string> compositeCentralStoreFileList = Utils::split(compositeFileList, ':');
+    std::vector<std::string>::iterator compositeCentralStoreFileIterator;
+    std::vector<std::string> availableCompositeFileList = Utils::split(availableFileList, ':');
+    std::vector<std::string>::iterator availableCompositeFileIterator;
+
+    for (availableCompositeFileIterator = availableCompositeFileList.begin(); availableCompositeFileIterator != availableCompositeFileList.end(); ++availableCompositeFileIterator) {
```

Comment:
Lets break the the lengthy lines into two

Reply:
Fixed this issue